ovs_performance.py

```diff
@@ -1295,28 +1295,33 @@ def test_p_zero_loss(stream_size_list, packet_size_list, **kwargs):
 #       modifiers. We could do a destination IP only OF
```

Comment:
What about this comment? What is still true, and what is not?

We also need some documentation on how to run this with both the Kernel and DPDK datapath (if supported).